### PR TITLE
Add repository containers docs link

### PR DESCRIPTION
**Pull request by bamurtaugh**
Based on https://github.com/microsoft/vscode-docs/pull/5198 and https://github.com/microsoft/vscode-remote-release/issues/6013

### CODE REVIEW

**Comment by Chuxel** [comment]
@chrmarti @bamurtaugh I think we should also remove reference to repository containers from the list here and possibly update the README under repository-containers to point to documentation on setting them up for yourself.   I think we'd consider these a legacy artifact at this point.

I don't believe these are supported by Codespaces either.

**Review by chrmarti — APPROVED** [review]
LGTM, thanks!

**Comment by bamurtaugh** [comment]
Thanks both for reviewing!

I updated the main readme to remove the mention of repository containers, and this readme links to further documentation (included in original PR, but I just made an update to the wording):

> You may review [further documentation](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) about repository containers to learn how to best configure them.

@Chuxel Do these updates sound reasonable to you?

File: repository-containers/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
 
-You may also review [further documentation](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) about repository containers.
+You may review [further documentation](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) about repository containers to learn how to best configure them.
 
-However, this is **not supported** from [GitHub Codespaces](https://github.com/features/codespaces) currently. 
+This is **not supported** from [GitHub Codespaces](https://github.com/features/codespaces) currently. 
 
 See [CONTRIBUTING.md](../CONTRIBUTING.md) for details on contributing a container.
```
Comment (Chuxel) [inline repository-containers/README.md:9]: Maybe we remove this and say that we are not accepting contributions for new repository containers and that we encourage you to use the Remote - Containers capability to have a local set of them instead?

Comment (bamurtaugh) [reply repository-containers/README.md]: Makes sense. Added:

> We are not accepting contributions for new repository containers. If you are interested in repository containers, we encourage you to use the [Remote - Containers capability](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) to have a local set of them.

Comment (Chuxel) [reply repository-containers/README.md]: LGTM!

**Review by Chuxel — APPROVED** [review]

**Pull request merged**